install-latest.sh

```diff
@@ -54,7 +54,8 @@ execute() {
       binexe="${binexe}.exe"
     fi
     log_debug "installing binary: $binexe"
-    cp "${srcdir}/${binexe}" "${BINDIR}/${binexe}" 2> /dev/null || sudo cp "${srcdir}/${binexe}" "${BINDIR}/"
+    # Use mv instead of cp, as copying when a binary already exists in that spot can create an invalid binary on macOS
```

Comment:
This is the only change in the PR. The rest of the changes are just removing whitespace